.github/workflows/pr.yml

```diff
@@ -0,0 +1,47 @@
+name: PR
+
+# Run daily and on PRs.
+on:
+  schedule:
+    - cron: '0 0 * * *'
```

Comment:
REQ: Why do we need this to run daily? If you remove the schedule it will run every time the PR is updated, which would make more sense to me.